libs/store/data/src/lib/root.ts

```diff
@@ -33,6 +39,8 @@ export const entitiesReducer = combineReducers({
   environments: environments,
   applications: applications,
   databases: databases,
+  environmentVariables: environmentVariables,
+  secretEnvironmentVariables: secretEnvironmentVariables,
```

Comment:
Why we didn't have group `variables` and inside we have this two elements? Because environmentVariable and secretEnvironment seems linked

Reply:
The thing is that they have two different property on the whole interface... It makes me make this difference for this sole reason. secrets have `override_secret` and `alias_secret` when public have `override_variable` and `alias_variable`